### PR TITLE
Add TEMPERATURELIMIT setting

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following:**

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [X] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#git-commit---signoff))

---

**What does this PR aim to accomplish?:**

Add a user-definable temperature limit instead of the hard-coded value last changed in https://github.com/pi-hole/AdminLTE/pull/293

**How does this PR accomplish the above?:**

Add `TEMPERATURE_LIMIT` setting in setupVars.conf for setting the level beyond which we show the temperature icon in red on the dashboard.

**What documentation changes (if any) are needed to support this PR?:**

Will be addressed in a follow-up PR to the docs repo once we merge this.

### CODE REVIEW

**Comment by pralor** [comment]
This pull request has been mentioned on **Pi-hole Userspace**. There might be relevant details there:

https://discourse.pi-hole.net/t/make-temperature-icon-threshold-a-user-configurable-setting/22566/4

**Comment by DL6ER** [comment]
Default settings:
Screenshot from 2019-08-20 14-05-52
Screenshot from 2019-08-20 14-05-41

`TEMPERATURELIMIT=55` already red below 60°C
Screenshot from 2019-08-20 14-06-39

`TEMPERATURELIMIT=65` still blue above 60°C
Screenshot from 2019-08-20 14-06-24

File: scripts/pi-hole/php/header.php [hunk 1]
```diff
@@ -60,6 +60,15 @@
         {
             $temperatureunit = $_POST["tempunit"];
         }
+        // Get user-defined temperature limit if set
+        if(isset($setupVars['TEMPERATURELIMIT']))
```
Comment (Mcat12) [inline scripts/pi-hole/php/header.php]: It would be more readable to use `TEMPERATURE_LIMIT` as the key.

Comment (DL6ER) [reply scripts/pi-hole/php/header.php]: `TEMPERATURELIMIT` was chosen to be similar to the already existing key `TEMPERATUREUNIT`. We can surely change it.

**Comment by pralor** [comment]
This pull request has been mentioned on **Pi-hole Userspace**. There might be relevant details there:

https://discourse.pi-hole.net/t/pi-hole-4-3-2-release-notes/23852/1

**Pull request merged**